sdk/containerapps/Azure.ResourceManager.Applications.Containers/src/autorest.md

```diff
@@ -7,7 +7,7 @@ Run `dotnet build /t:GenerateCode` to generate code.
 azure-arm: true
 csharp: true
 library-name: containerapp
-namespace: Azure.ResourceManager.containerapp
```

Comment:
Do we have a reason that previously the namespace here is different from its real namespace?

Reply:
good catch, I forgot to fix it as the namespace was final approved before I initial this RP